ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AttestationManager.java

```diff
@@ -141,12 +130,12 @@ private void processAttestation(final DelayableAttestation delayableAttestation)
   @Override
   protected SafeFuture<?> doStart() {
     eventBus.register(this);
-    return this.pendingAttestations.start();
+    return SafeFuture.completedFuture(null);
```

Comment:
I think you can use:
```suggestion
    return SafeFuture.COMPLETE;
```

Reply:
Done.